src/index.html

```diff
@@ -7,8 +7,50 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <span class="logo">
+        <a href="#home"><img src="images/logo.png" alt="logo"></a>
```

Comment:
```suggestion
        <a href="#home">
          <img src="images/logo.png" alt="logo">
        </a>
```

writing clean code is important. refactor it in the rest of your code